examples/complete-vpc/main.tf

```diff
@@ -19,18 +24,35 @@ module "vpc" {
 
   create_database_subnet_group = false
 
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
   enable_nat_gateway = true
   single_nat_gateway = true
 
   enable_vpn_gateway = true
 
-  enable_s3_endpoint       = true
-  enable_dynamodb_endpoint = true
-
   enable_dhcp_options              = true
   dhcp_options_domain_name         = "service.consul"
   dhcp_options_domain_name_servers = ["127.0.0.1", "10.10.0.2"]
 
+  # VPC endpoint for S3
+  enable_s3_endpoint = true
+
+  # VPC endpoint for DynamoDB
+  enable_dynamodb_endpoint = true
+
+  # VPC endpoint for SSM
+  enable_ssm_endpoint              = true
+  ssm_endpoint_private_dns_enabled = true
+  ssm_endpoint_security_group_ids  = ["${data.aws_security_group.default.id}"]
+
+  //  ssm_endpoint_subnet_ids = ["..."]
```

Comment:
@tbugfinder I've added argument `ssm_endpoint_subnet_ids` which can be specified to make sure that the list of subnets contains at most just one subnet per AZ. I don't know any easy way to filter values another way automatically.